editor-guidelines.md

```diff
@@ -198,6 +198,47 @@ Upload the original image to the [gallery/originals](https://github.com/programm
 
 Add the image to the lessons page by editing the HTML in [lessons/index.md](https://github.com/programminghistorian/jekyll/blob/gh-pages/lessons/index.md) so that the image source matches the new image’s file location. In our previous example, the HTML line would be: `<img src="../gallery/cleaning-data-with-openrefine.png">`. The new image should appear on the [Lessons page]( http://programminghistorian.org/lessons)!
 
+### 8) Confirm all links are functioning correctly
+
+Once you push your changes on to the `gh-pages` branch of the [programminghistorian][ph_repo] repository, the site will be automatically tested by [Travis CI] ([Continuous Integration]).
+This test process checks two things: first, that all YAML and markdown code is parseable, and second, that all the hyperlinks on the site point to valid, operational pages.
+
+[ph_repo]: https://github.com/programminghistorian/jekyll
+
+[Travis CI]: https://travis-ci.org
+
+[Continuous Integration]: https://www.thoughtworks.com/continuous-integration
+
+We run these builds primarily to check that URLs that _once_ were functional are _still_ functional, as often times external web pages are moved to new addresses, or are no longer live.
+They are also an excellent way to catch small typos that may have evaded authors, editors, and reviewers.
+The status of these tests (often called a "Build Status" on Travis CI and on GitHub) can be seen by navigating to the [programminghistorian repository page][ph_repo], and clicking "Commits" on the upper left side of the code menu.
+
+![GitHub commit menu location](images/editor-guidelines/gh_commits_location_screen.png)
+
+This will show you the list of every change made to the main repository, along with a status icon:
+
+- Green check mark: you're good to go! All the links on the page were checked and found valid. **You may skip the rest of this section.**
```

Comment:
Quick thought: what about an internal link to the end of the section inside the "You may skip the rest of this section"?